### PR TITLE
new chart stable/gce-ingress

### PR DESCRIPTION
**Pull request by paulczar**
This chart implements gce-ingress for non-gke clusters that are running
on GCE or otherwise want to utilize GCE load balancing services for
Ingress resources.

Signed-off-by: Paul Czarkowski <username.taken@gmail.com>

**What this PR does / why we need it**:

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: stable/gce-ingress/templates/deployment-backend.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
+apiVersion: apps/v1beta2
```
Comment (davidkarlsen) [inline stable/gce-ingress/templates/deployment-backend.yaml]: use v1 api

File: stable/gce-ingress/templates/deployment-controller.yaml [hunk 2]
```diff
@@ -0,0 +1,84 @@
+apiVersion: apps/v1beta2
```
Comment (davidkarlsen) [inline stable/gce-ingress/templates/deployment-controller.yaml]: use v1 api

File: stable/gce-ingress/templates/NOTES.txt [hunk 3]
```diff
@@ -0,0 +1,51 @@
+GCE Ingress has been deployed.  You should be able to create Ingress resources.
+
+Example:
+```yaml
+apiVersion: extensions/v1beta1
```
Comment (davidkarlsen) [inline stable/gce-ingress/templates/NOTES.txt]: use v1 api

File: stable/gce-ingress/templates/rbac.yaml [hunk 4]
```diff
@@ -0,0 +1,34 @@
+{{ if .Values.rbac.enabled -}}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ include "gce-ingress.fullname" . }}
```
Comment (davidkarlsen) [inline stable/gce-ingress/templates/rbac.yaml]: see https://github.com/helm/helm/blob/master/docs/chart_best_practices/rbac.md - use a helper for the serviceaccount name

**Review by davidkarlsen — CHANGES_REQUESTED** [review]

File: stable/gce-ingress/templates/deployment-backend.yaml (hunk 1 above)
Comment (paulczar) [reply stable/gce-ingress/templates/deployment-backend.yaml]: ahhh good catch!  the `helm create` templates must be wayyyy behind.

File: stable/gce-ingress/templates/rbac.yaml (hunk 4 above)
Comment (paulczar) [reply stable/gce-ingress/templates/rbac.yaml]: wicked thanks!

**Comment by paulczar** [comment]
@davidkarlsen can I get a re-review now that I've addressed your concerns ?

**Review by davidkarlsen — APPROVED** [review]

**Pull request merged**